src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ObjectTypeVisitor.cs

```diff
@@ -19,6 +19,18 @@ namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Visitors
     /// </summary>
     public class ObjectTypeVisitor : TypeVisitor
     {
+        private readonly HashSet<Type> _noVisitableTypes = new HashSet<Type>
+        {
+            typeof(Guid),       typeof(DateTime),       typeof(DateTimeOffset),
+            typeof(Uri),        typeof(Type),           typeof(object),
+            typeof(byte[])
+        };
+
+        private readonly HashSet<string> _noAddedKeys = new HashSet<string>
+        {
+            "object", "jToken", "jObject"
```

Comment:
Can we use all lowercased or UPPERCASED here to avoid case confusion??

Reply:
I used UPPERCASE for the key, thanks